src/libServer/JSONConversion.cpp

```diff
@@ -421,14 +435,9 @@ Address JSONConversion::checkJsonGetEthCall(const Json::Value& _json) {
     throw jsonrpc::JsonRpcException(Server::RPC_INVALID_PARAMETER,
                                     "must contain toAddr");
   }
-  string lower_case_addr;
-  if (!AddressChecksum::VerifyChecksumAddress(_json["toAddr"].asString(),
-                                              lower_case_addr)) {
-    LOG_GENERAL(INFO,
-                "To Address checksum wrong " << _json["toAddr"].asString());
-    throw jsonrpc::JsonRpcException(Server::RPC_INVALID_PARAMETER,
-                                    "To Addr checksum wrong");
-  }
+
+  string lower_case_addr = boost::to_lower_copy(_json["toAddr"].asString());
```

Comment:
@KaustubhShamshery is it ok to forgo the address checksum check for this call? It seems that it's not technically neccessary

Reply:
Well might wanna check if the address contains "0x" and remove it if does not

Reply:
Do you mean if it contains 0x you should strip it off, right?

Reply:
Well, @n-hutton did you change this as advised by Kaustubh? It broke the tests (which don't run on CI yet :))

Reply:
no - it should be fixed in this PR though https://github.com/Zilliqa/Zilliqa/pull/2828